openedx/features/pakx/lms/overrides/views.py

```diff
@@ -42,8 +41,13 @@ def courses(request):
 
     add_course_progress_and_resume_info_tags_to_enrolled_courses(request, courses_list)
 
+    show_only_enrolled_courses = waffle.switch_is_active('show_only_enrolled_courses')
+
     for course in courses_list:
-        if course.enrolled and course.user_progress == '100':
+        if show_only_enrolled_courses:
```

Comment:
combine both if statements 
`if show_only_enrolled_courses and not course.enrolled: `